index.js

```diff
@@ -14,9 +14,9 @@
 const Task = require('./lib/task');
 const Queue = require('./lib/queue');
 
-async function enqueue(task, args, { name = 'default' } = {}) {
+async function enqueue(task, args, { name = 'default', title = undefined } = {}) {
```

Comment:
@tomkuk you don't need to set it to `undefined`, uninitialized variables are set this way by default